digit-health-services/libraries/health-services-common/settings.xml

```diff
@@ -2,9 +2,16 @@
 <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
+    <mirrors>
+        <mirror>
+            <id>nexus</id>
+            <mirrorOf>central</mirrorOf>
+            <url>https://nexus-repo.egovernments.org/nexus/content/groups/public/</url>
```

Comment:
Do we need to change the URL here as well?

Reply:
Yes, Mirror section is used for downloading jar/dependencies from nexus.

Reply:
Removing <mirrors> tag because it is not required.